src/gfn/modules.py

```diff
@@ -306,6 +394,13 @@ def __init__(
             preprocessor=preprocessor,
             is_backward=is_backward,
         )
+        reduction_fxns = {
+            "mean": torch.mean,
+            "sum": torch.sum,
+            "prod": torch.prod,
+        }
```

Comment:
here you can use the global constant, if you follow the previous comment

Reply:
Thanks - I've done this